task/pylint/0.2/pylint.yaml

```
@@ -0,0 +1,34 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: pylint
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: python, pylint
+    tekton.dev/displayName: pylint
+
+spec:
+  description: >-
+    This task will run pylint on the provided input.
+
+  workspaces:
+    - name: source
+  params:
+    - name: image
+      description: The container image with pylint
+      default: docker.io/cytopia/pylint@sha256:6e5b49b6d54cbd845b93139eeb3f4f558b07bf6a87001ce254782463df1443d2
+    - name: args
+      description: The arguments to pass to the pylint CLI.
+      type: array
+      default: ["--help"]
```

I would keep here a separat parameter for the directory or the path to the module as without this pylint is not working. The current setup would require to specify the directory always as an argument, so to run it either need to write ["."] or ["-r", "y", "src"] like in the test. This additional parameter would make it clearer for me personally and I also think results in less confusion.
```suggestion
    - name: args
      description: The arguments to pass to the pylint CLI.
      type: array
      default: ["--help"]
    - path
      description: The path to the module which should be analysed by pylint
      default: "."
```